exercises/practice/equilateral_triangle/equilateral_triangle.py

```diff
@@ -0,0 +1,21 @@
+from z3 import *
+
+def generate_third_points(first_point, second_point):
```

Comment:
I would add a comment here indicating that this is where the user should add their code (you can use the same one that I have in mine to be consistent if you want).